.github/workflows/build.yaml

```
@@ -23,6 +23,16 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements.txt
+      - name: Debug Environment Variables
+        run: |
+          echo "MONGO_DB_NAME is set to: $MONGO_DB_NAME"
+        env:
+          MONGO_DB_NAME: ${{ secrets.MONGO_DB_NAME }}
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)